gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/CompatibilityAttributeFilterElements.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2007-2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.executeafm.afm;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+import java.util.List;
+
+import static com.fasterxml.jackson.databind.JsonMappingException.from;
+
+@Deprecated
```

Comment:
plz add some javadoc, why it's here and when can we remove it

Reply:
ups, this class is not needed in final solution - will delete it